hackathon_site/event/models.py

```diff
@@ -1,8 +1,9 @@
 from django.db import models
-from django.contrib.auth.models import User
-
+from django.contrib.auth import get_user_model
 import uuid
 
+User = get_user_model()
```

Comment:
Why do we need to do it this way and not just import the class?

Reply:
In Django, it's possible to override the default user model: https://docs.djangoproject.com/en/3.0/topics/auth/customizing/#referencing-the-user-model

Using `get_user_model()` instead of `from django.contrib.auth.models import User` lets forks of this repo specify a custom user model in the settings before they first migrate. The migration file was smart enough to reference `settings.AUTH_USER_MODEL` to begin with, so no changes are needed there: https://github.com/ieeeuoft/hackathon-template/blob/develop/hackathon_site/event/migrations/0001_initial.py#L78.

We'll need to change this elsewhere like in the `applications` app.